src/tests/test_meta_blk_mgr.cpp

```diff
@@ -579,24 +582,28 @@ class VMetaBlkMgrTest : public ::testing::Test {
         m_update_cnt = 0;
         m_rm_cnt = 0;
         m_total_wrt_sz = 0;
+        m_restart_cnt = 0;
     }
 
     void register_client() {
         m_mbm = &(meta_service());
         m_total_wrt_sz = m_mbm->used_size();
-
         HS_REL_ASSERT_EQ(m_mbm->total_size() - m_total_wrt_sz, m_mbm->available_blks() * m_mbm->block_size());
-
         m_mbm->deregister_handler(mtype);
         m_mbm->register_handler(
             mtype,
             [this](meta_blk* mblk, sisl::byte_view buf, size_t size) {
                 if (mblk) {
                     std::unique_lock< std::mutex > lg{m_mtx};
-                    m_cb_blks[mblk->hdr.h.bid.to_integer()] = md5_sum(r_cast< const char* >(buf.bytes()), size);
+                    m_cb_blks[mblk->hdr.h.bid.to_integer()] = sb_info_t {
+                        mblk, md5_sum(r_cast< const char* >(buf.bytes()), size)};
                 }
             },
-            [this](bool success) { HS_DBG_ASSERT_EQ(success, true); });
+            [this](bool success) {
+                HS_DBG_ASSERT_EQ(success, true);
+                m_total_wrt_sz = m_mbm->used_size();
+                HS_REL_ASSERT_EQ(m_mbm->total_size() - m_total_wrt_sz, m_mbm->available_blks() * m_mbm->block_size());
```

Comment:
update the `m_total_wrt_sz` at the end of recovery.   it was set to 0 in L590 before MetaService recovered.